tokio/src/lib.rs

```diff
@@ -44,7 +44,7 @@
 //! [signal]: crate::signal
 //! [fs]: crate::fs
 //! [runtime]: crate::runtime
-//! [website]: https://tokio.rs/docs/overview/
+//! [website]: crate
```

Comment:
I don't think this link should be changed.

Reply:
Agree, thanks :)

Reply:
fixed, thanks!